### ISSUE TITLE
Discovery - Overall DD monitor for all services failing

### ISSUE THREAD

**Issue by luis-simauchi** [department-of-veterans-affairs/va.gov-team#78718]
lilhan mentioned there might be a need to have one more monitor for the overall services on the 1010 forms to measure weather there is a bigger problem

**Comment by hdjustice** [comment]
Build a monitor showing all services, so that we can see a bigger picture, in the case of catastrophic events

**Comment by luis-simauchi** [comment]
yep we took care of this a while ago, this ticket was probably a duplicate as we broke up the initial ticket into smaller by form tickets
